### ISSUE TITLE
Email/FreshDesk Handler?

### ISSUE THREAD

**Issue by GalGreenfield** [snowflakedb/SnowAlert#388]
Hello,

Does SnowAlert have an email handler, or alternatively a FreshDesk handler?

We've migrating our issues handling to FreshDesk and would like to create tickets there, whether directly or via email (which FreshDesk will use to create tickets automatically).

**Comment by veraobula** [comment]
Hi, Any update regarding SnowAlert & email handler integration

**Comment by sfc-gh-afedorov** [comment]
Yes, while the rest of these docs is a WIP, the SMTP one is documented [here](https://docs.snowalert.com/alerts/handlers/smtp).
